CRM/Core/BAO/Mapping.php

```diff
@@ -1222,67 +1296,15 @@ public static function saveMappingFields(&$params, $mappingId) {
     foreach ($params['mapper'] as $key => $value) {
       $colCnt = 0;
       foreach ($value as $k => $v) {
+        $locationTypeid = NULL;
+        $phoneTypeid = NULL;
```

Comment:
These 2 vars appear unused.

Reply:
but they protect from enotices

Reply:
I read the code again and still don't see how they are doing anything in this spot. I see how variables with the same name are used in the new function, but not here.

Reply:
ah maybe it was a refactor ago - push a change if you want to merge it now - I'm off to bed. The tests were failing on the enotice originally so it passes tests then all good